examples/styles.scss

```diff
@@ -79,6 +79,10 @@ pre {
 	margin: 15px auto;
 }
 
+.invisible_skin {
+	display: none !important;
```

Comment:
https://github.com/Wikia/app/blob/88070491574e2ea5c754957f909ff25c6e60ea46/skins/oasis/css/core/ads.scss#L28
FYI: It's a production copied hack :o